### ISSUE TITLE
FDR & Annoataions

### ISSUE THREAD

**Issue by monprin** [LinkageIO/cob#49]
Going to add as part of a Camoco to handle all annotations of various types. Removes reliance on other data structures and configuration/installation easier. GWS will still handle running the algorithm to do keyword enrichment, but needs to be revamped to support cob objects. More stable in the long term.

**Comment by schae234** [comment]
I dont think GWS will be necessary here since we aren't doing enrichment. All the annotation piece will do is link external tables to the genes that are generated from SNP to gene mapping. Like this [code](https://github.com/schae234/Camoco/blob/04f079e9ee2648132550bd24b7b05d3d9c956c6e/camoco/cli/commands/snp2gene.py#L69-L82).

**Comment by monprin** [comment]
Well no, not for the GWAS Data, but for the functional annotations it can still be used, plus at the moment, it is still the engine powering the functional Annotation Database (though I would like to change that and make a functional annotation engine in Camoco directly).

**Comment by monprin** [comment]
I have built the GWASData Engine and I think it is pretty spiffy, I will let you know when I post the code so you can take a look, still have to test it out and integrate it into the site. I split it entirely from functional annotations since it didn't really seem to be a happy fit.

**Comment by schae234** [comment]
OK, :fire_engine: :fire: :fire: 
Sounds good!

**Comment by monprin** [comment]
FDR and the new functional annotations engine are on Github (not live on lovelace yet though) and integrated into the COB code, so we are back to it being completely and solely integrated into Camoco.

The two classes I added to support this can be seen [here](https://github.com/monprin/Camoco/blob/master/camoco/Annotation.py), but given your go ahead, I would like to just integrate them into their respective parents. I designed both of them with a mind towards that, so they more or less follow the standards for those parts of the code. If you're okay with it, I can do it after the beta meeting. 

Also, I put in some ground work to support ortholog functional annotations, is that something you think would be helpful? (They would be kept separate from the primary annotations)

Let me know if you have any thoughts.
